build-system/tasks/runtime-test/index.js

```diff
@@ -413,7 +432,7 @@ async function runTests() {
 
     if (browsers.beta.length) {
       const allBatchesExitCodes = await runTestInBatchesWithBrowsers(
-          'beta', browsers.beta);
+          'beta', browsers.beta, /* runComplete */ () => {});
```

Comment:
Nit: do you want to add the 260 beta tests to the sum? That's a total 650 tests which is pretty cool

Reply:
@danielrozenberg is going to generate a separate non-blocking report for tests on beta browsers.

Reply:
Yep. Right now reporting even one failed tests fails the whole shard. I need to modify the test-status app to allow reporting "flaky" test types (for saucelabs-beta and for e2e)